apps/microsoft-teams/app-actions/src/services/msteams-bot-service.ts

```diff
@@ -0,0 +1,32 @@
+import { EntryActivityMessage, SendMessageResult } from '../types';
+
+export class MsTeamsBotService {
+  constructor(public readonly botServiceUrl: string, public readonly apiKey: string) {}
+
+  async sendEntryActivityMessage(
+    entryActivitytMessage: EntryActivityMessage,
```

Comment:
just a small typo in this parameter -- `entryActivitytMessage ` should be `entryActivityMessage `